ec2utils/ec2uploadimg/ec2uploadimg

```diff
@@ -145,6 +145,12 @@ argparse.add_argument(
     help='AWS secret access key (Optional)',
     metavar='AWS_SECRET_KEY'
 )
+argparse.add_argument(
+    '-o', '--session_token',
```

Comment:
@rjschwei not sure about the ``-o`` though. ``-s`` and ``-t`` is already in use.